File: 3019-FurthestPointFromOrigin/3019-FurthestPointFromOrigin.py
# Last updated: 15/08/2026, 19:50:54
class Solution:
    def furthestDistanceFromOrigin(self, moves: str) -> int:
        count=0
        ans=0
        for ch in moves:
            if ch=='L':
                ans-=1
            elif ch=='R':
                ans+=1
            else:
                count+=1
        
        return abs(ans)+count



        

File: 3019-FurthestPointFromOrigin/3019-FurthestPointFromOrigin.java
// Last updated: 15/08/2026, 19:51:00
class Solution {
    public int furthestDistanceFromOrigin(String moves) {
        int count=0;
        int ans=0;
        for(char ch:moves.toCharArray())
        {
            if(ch=='L')
            ans--;
            else if(ch=='R')
            ans++;
            else
            count++;
        }
        return Math.abs(ans)+count;
    }
}